udata/core/dataset/models.py

```diff
@@ -206,7 +206,7 @@ def clean(self):
     @property
     def closed_format(self):
         """Return True if the specified format is in CLOSED_FORMATS."""
-        return self.format.lower() in CLOSED_FORMATS
+        return self.format and self.format.lower() in CLOSED_FORMATS
```

Comment:
So if format is not defined we consider that this is an open format :thinking: 
Maybe the opposite behavior makes more sense (ie. consider undefined format as closed format by default and so force user to define format if they don't want anymore the quality warning)